internal/manager/config.go

```diff
@@ -181,7 +181,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.KongWorkspace, "kong-workspace", "", "Kong Enterprise workspace to configure. Leave this empty if not using Kong workspaces.")
 	flagSet.BoolVar(&c.AnonymousReports, "anonymous-reports", true, `Send anonymized usage data to help improve Kong.`)
 	flagSet.BoolVar(&c.EnableReverseSync, "enable-reverse-sync", false, `Send configuration to Kong even if the configuration checksum has not changed since previous update.`)
-	flagSet.BoolVar(&c.UseLastValidConfigForFallback, "use-last-valid-config-for-fallback", false, `When recovering from config push failures, use the last valid configuration cache to backfill broken objects.`)
+	flagSet.BoolVar(&c.UseLastValidConfigForFallback, "use-last-valid-config-for-fallback", false, fmt.Sprintf(`When recovering from config push failures, use the last valid configuration cache to backfill broken objects. Can only be used with the %s feature gate enabled.`, featuregates.FallbackConfiguration))
```

Comment:
~Nit: this will get out of date when we remove the feature gate. Perhaps we can use wording that just references whether the feature is enabled?~

Actually it occurred to me that there's no way to disable the fallback config when we get rid of the feature flag in the future. Was that considered? Users might want to keep the more deterministic/stateless (almost, with the asterisk of last known config) nature when we graduate fallback config. 🤔

Reply:
From what I hear is that we should have one more CLI flag to enable the fallback config feature so that after we graduate `FallbackConfiguration` feature _gate_ (making it default true) we still allow users to disable the fallback behavior.

I think that once we graduate `FallbackConfiguration`, we can add a `--disable-fallback-configuration` (default false) that will still allow users to disable it.

Reply:
@mheap WDYT? ☝️

Reply:
Makes sense to me